scripts/render-notebooks.R

```diff
@@ -27,48 +31,43 @@ option_list <- list(
 # Parse options
 opt <- parse_args(OptionParser(option_list = option_list))
 
-# Make it normalized to root_dir
-normalized_bib_file <- normalizePath(opt$bib_file, mustWork = TRUE)
+# Make these file paths
+opt$bib_file <- file.path(opt$bib_file)
+opt$rmd <- file.path(opt$rmd)
+
+# Check that the file exists
+if (!file.exists(opt$rmd)) {
+  stop("Rmd file specified with --rmd is not found.")
+}
 
-# Check that the file exists in the root directory
-if (!file.exists(normalized_bib_file)) {
-  message(normalized_bib_file)
+# Check that the file exists
+if (!file.exists(opt$bib_file)) {
   stop("File specified for --bib_file option is not at the specified file path.")
 } else {
   bib_specify_line <- paste("bibliography:", normalized_bib_file)
```

Comment:
You probably want to get the full path to avoid `.Rmd` path weirdness here.

```suggestion
  bib_specify_line <- paste("bibliography:", normalizePath(opt$bib_file))
```